haskell/haskell.bzl

```diff
@@ -126,7 +126,14 @@ def _mk_binary_rule(**kwargs):
       main_file = attr.label(
         allow_single_file = FileType([".hs", ".hsc", ".lhs"]),
         doc = "File containing `Main` module.",
-      )
+      ),
+      _dummy_static_lib = attr.label(
+        default = Label("//haskell:dummy_static_lib"),
```

Comment:
I _think_, but I'm not sure, that this needs to be a fully qualified label, in the style of https://github.com/tweag/rules_haskell/pull/343/files#diff-f479d5c4fb32903b900b580306254f64R89, to work correctly in downstream repos that import rules_haskell as an external repo.

Reply:
Ah, because they might see it as `//` of their project. I think @guibou mentioned something along those lines.